src/commands/lambda/flare.ts

```diff
@@ -166,8 +170,58 @@ export class LambdaFlareCommand extends Command {
     const configStr = util.inspect(config, false, undefined, true)
     this.context.stdout.write(`\n${configStr}\n`)
 
+    // Get project files
+    this.context.stdout.write(chalk.bold('\n📁 Searching for project files in current directory...\n'))
+    const projectFilesToPath = await getProjectFiles()
+    let projectFilesMessage = chalk.bold('\n✅ Found project file(s):\n')
+    if (projectFilesToPath.size === 0) {
+      projectFilesMessage = commonRenderer.renderSoftWarning('No project files found.')
+    }
+    this.context.stdout.write(projectFilesMessage)
+    for (const projectFilePath of projectFilesToPath.values()) {
+      this.context.stdout.write(`• ${projectFilePath}\n`)
+    }
+
+    // Additional files
+    this.context.stdout.write('\n')
+    const additionalFiles = new Set<string>()
+    const addFilesQuestion = await inquirer.prompt(
+      confirmationQuestion('Do you want to specify any additional files to send to Datadog support?')
+    )
+    while (addFilesQuestion.confirmation) {
+      this.context.stdout.write('\n')
+      let filePath: string
+      try {
+        filePath = await requestFilePath()
```

Comment:
Here you are properly creating a prompt for the file, but not for the question above.